docs/DeveloperGuide.md

```diff
@@ -349,39 +349,55 @@ The following activity diagram summarizes what happens when a user executes the
 ## **Appendix: Requirements**
 
 ### Product scope
+**Target user profile story**: <br> <br>
+Ian is a former air crew with Singapore Airlines. He recently lost his job due to the COVID-19 pandemic resulting in the collapse in international travel.
+He found a job with the University Health Center (UHC) as an admin executive. He is a fast typer with good knowledge of the command line interface.
+Currently, Ian is using Excel to keep track of all the student records and vaccination appointments in UHC. With so many rows and columns of data to look at, 
+Ian finds it difficult to sieve information and is prone to making human errors.
```

Comment:
Should it be "and prone to making human errors," coz it's a continuation of "finds it ... prone to..."

Reply:
I don't think 'finds it prone to making human errors' is correct English though, so it should be "finds it difficult..." and "is prone to..."